src/cmd/package.go

```diff
@@ -392,14 +413,86 @@ func (o *packageInspectOptions) run(cmd *cobra.Command, args []string) error {
 	return definitionOpts.run(cmd, args)
 }
 
-// PackageInspectSBOMOptions holds the command-line options for 'package inspect sbom' sub-command.
-type PackageInspectSBOMOptions struct {
+type packageInspectManifestsOpts struct {
+	skipSignatureValidation bool
+	components              string
```

Comment:
Interesting to see that the `components` flag is simply a string versus a slice of strings. Looks to be an existing pattern - but surprised me nonetheless given the `StringSliceVar` flag options. (No change required)

Reply:
Yeah and it seems like we can actually use StringSliceVar without it being a breaking change. If I were to guess the name reason it's a slice is because you can do globbing - `Globbing component names with '*' and deselecting 'default' components with a leading '-' are also supported`.

Reply:
I guess either way even if you are using globbing it's still a list, trying to decide whether or not I should make an issue for this

Reply:
It's quite superfluous - I don't think there is any criticality so I'd argue no.